src/DagDerivedCompiler.cpp

```diff
@@ -240,27 +242,42 @@ struct CompileDagDerivedWorker
     }
 
 
+    void PrintStats()
+    {
+        uint64_t totalFlattenedEdges = 0;
+        uint64_t toBuildEdges = 0;
+        uint64_t toUseEdges = 0;
+
+        for (int32_t i = 0; i < node_count; ++i)
+        {
+             totalFlattenedEdges += combinedDependenciesBuffers[i].GetCount();
+             toBuildEdges += dag->m_DagNodes[i].m_ToBuildDependencies.GetCount();
+             toUseEdges += dag->m_DagNodes[i].m_ToUseDependencies.GetCount();
+        }
+
+        printf("Baked DAG. NodeCount=%d, TotalFlattenedEdges=%" PRId64 " ToBuildEdges=%" PRId64 " ToUseEdges=%" PRId64 " MaxPoints=%d\n", node_count, totalFlattenedEdges, toBuildEdges, toUseEdges, this->max_points);
+    }
 
     bool WriteStreams(const char* dagderived_filename)
     {
         MemAllocLinearScope scratchScope(scratch);
 
         combinedDependenciesBuffers = HeapAllocateArrayZeroed<Buffer<int32_t>>(heap, node_count);
-        for (int32_t i = 0; i < node_count; ++i)
-        {
-            for(int dep : dag->m_DagNodes[i].m_ToBuildDependencies)
+            for (int32_t i = 0; i < node_count; ++i)
```

Comment:
indentation seems off here, and for some of the other new for loops

Reply:
fixed